src/molecules/product-grid-item/ProductGridItem.stories.js

```diff
@@ -0,0 +1,157 @@
+import { addDecorator, storiesOf } from '@storybook/vue'
+import { action } from '@storybook/addon-actions'
+import { text, number, object } from '@storybook/addon-knobs'
+import StoryRouter from 'storybook-vue-router'
+
+import AProductGridItem from './ProductGridItem.vue'
+import AIcon from '../../atoms/icon/Icon.vue'
+import ABadge from '../../atoms/badge/Badge.vue'
+import ARating from '../../molecules/rating/Rating.vue'
+
+const defaultPrices = {
+  price: '$1684.00',
+  specialPrice: '$100.00',
+  oldPrice: '$299.99'
+}
+const defaultData = {
+  methods: {
+    addToCart: action('Added to Cart'),
+    addToWishList: action('Added to Wishlist'),
+    addToCompare: action('Added to Compare')
+  }
+}
+
+addDecorator(StoryRouter())
+storiesOf('Molecules/Product grid item', module)
```

Comment:
change to `Molecules/Product Grid Item`, the same as other components in develop

Reply:
no changes here

Reply:
I've changed